src/identity/Identity.ts

```diff
@@ -100,20 +192,101 @@ export default class Identity extends PublicIdentity {
 
   private readonly signKeyringPair: KeyringPair
   private readonly boxKeyPair: BoxKeyPair
+  /**
+   * @description Creates a new instance of public identity.
```

Comment:
* `getPublicIdentity()` doesn't create a new instance. It returns the public identity
* Maybe explain why this is the public identity

Reply:
I changed the wording and cleared it up a bit.